ddtrace/contrib/internal/rq/patch.py

```diff
@@ -13,21 +11,26 @@
 
 from ....ext import SpanKind
 from ....ext import SpanTypes
+from ....settings._config import _get_config
```

Comment:
Should we just directly import it from `_core` instead?
```suggestion
from ....settings._core import get_config as _get_config
```